_includes/post.json

```diff
@@ -0,0 +1,8 @@
+{
+  "title"    : "{{ post.title }}",
+  "category" : "{{ post.category }}",
+  "tags"     : [{% for tag in post.tags %}"{{ tag }}",{% endfor %} null],
+  "url"     : "{{ post.url }}",
+  "title_image" : "{{ post.title_image }}",
+  "prep_time" : "{{ post.prep_time }}"
+}
```

Comment:
"{{ post.title | xml_escape }}" seems like an easy way to do that for generating the json. I'm not sure the nicest way to convert it back to html for displaying - if the search_results_list could be parsed with jekyll with markdownify, but handlebars helpers are more work.